couchrest_model.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency("couchrest",   "2.0.0.rc3")
-  s.add_dependency("activemodel", "~> 5.1.5")
+  s.add_dependency("activemodel", "~> 5.2.1")
```

Comment:
We could also do: `s.add_dependency("activemodel", ">= 5.1")` to avoid problems in the future :)

Reply:
@pinoss sure